### PR TITLE
Remove *-UserAuthentication commands

### PR DESCRIPTION
**Pull request by andres-canello**
Remove *-UserAuthenticationMethod commands from the Identity.SignIns module.

### CODE REVIEW

File: src/Identity.SignIns/Identity.SignIns/readme.md [hunk 1]
```diff
@@ -48,6 +48,10 @@ directive:
       subject: ^(DataPolicyOperation)(\1)+
     set:
       subject: $1
+# Remove cmdlets
+  - where:
+      subject: ^UserAuthenticationMethod$
```
Comment (peombwa) [inline src/Identity.SignIns/Identity.SignIns/readme.md:53]:
```suggestion
      subject: ^UserAuthentication$
```
My bad, the subject regex should be `^UserAuthentication$`.

**Review by peombwa — APPROVED** [review]

**Comment by ddyett** [comment]
@peombwa is there an issue for the reason this needed to be removed?

**Comment by andres-canello** [comment]
> @peombwa is there an issue for the reason this needed to be removed?

@ddyett We don't have any methods associated to the property "Authentication", this is why I'm asking to remove the associated commands.

**Comment by peombwa** [comment]
@ddyett `Get-MgUserAuthentication` currently calls `https://graph.microsoft.com/beta/users/{user-id}/authentication` which is an invalid path and return a 404. The commands should be `*-MgUserAuthentication*`.

**Pull request merged**